[wip][redesign] Add admin pages and remaining proposal features

fernandoabolafio opened the pull request:
This PR adds:
- [x] Admin listing of proposals
- [ ] User listing of proposals
- [ ] List pagination for admin and user listing
- [x] Proposal unvetted actions (censor, publish)
- [x] Proposal public actions (abandon, start vote, revoke vote, authorize vote)

lukebp commented:
1. When you censor a proposal, it removes it from the "Unreviewed" list but it doesn't add it to the "Censored" list until you refresh.

2. When I click on the saved draft proposal title I get a 404.  Clicking on the edit icon works as expected but not when you click the title.

MariaPleshkova commented:
Well, these are what I found:

1. We can switch to an admin view by clicking on "admin", but when we are already there we still see "admin" in drop down. I think we have to change it to "Public proposals" or just "Proposals" or smth like that and going back to Public proposals page by clicking on it
![image](https://user-images.githubusercontent.com/8841018/63528272-52719b80-c50b-11e9-8064-0c160516af23.png)

spec:
![image](https://user-images.githubusercontent.com/8841018/63528928-80a3ab00-c50c-11e9-8316-9c4453504b40.png)


2. It seems to me that the name of a user is closer to the arrow on my designs. Or proposals credits and a plus are farther from it. I mean right now it looks like there are 3 different elements:
![image](https://user-images.githubusercontent.com/8841018/63529356-2fe08200-c50d-11e9-83f0-c8c6118715cc.png)
But it has to be like this: 
![image](https://user-images.githubusercontent.com/8841018/63529395-41298e80-c50d-11e9-8687-528040f0fb95.png)
Hope I was clear:)

3. We have to add a hover effect to the arrow (that is next to the name in header). Let's make it our main blue color on hover #2970FF ( we already have the same pattern for cross icons on pop ups) 

4. Need to move this closer to the left side
![image](https://user-images.githubusercontent.com/8841018/63530109-7c788d00-c50e-11e9-9903-a6624c8faeff.png)

spec:
![image](https://user-images.githubusercontent.com/8841018/63530178-a0d46980-c50e-11e9-98d8-bba755a002a3.png)

5. I just also thought, maybe it would be better to add one more pop up to appear after clicking on Confirm button in Confirm/Report as spam pop ups? A pop up with "Confirmed"/"Reported as spam" text to give users some feedback that the action has been done successfully. What do you think?

fernandoabolafio commented:
@MariaPleshkova thanks for reviewing! I addressed all of your comments, except by the `#4`. I checked and all error messages have left padding of 20px as from the specs. 
About 5, I came up with this to give feedback on proposal actions:
![uifeedback](https://user-images.githubusercontent.com/14864439/63578237-5267af00-c590-11e9-9331-b5a32a417a4c.gif)

All changes are deployed to pi-redesign.

MariaPleshkova commented:
@fernandoabolafio this is what I've ended up with (the copy was just created by me, so we can change it, if it's not ok)
![image](https://user-images.githubusercontent.com/8841018/63652796-058cff80-c76d-11e9-8507-a75c4222b7ca.png)

spec: https://xd.adobe.com/spec/4e7eb732-a445-46b0-4f7e-4152097a10c8-384f/

fernandoabolafio commented:
@MariaPleshkova updated the actions feedback accordingly to your specs.

MariaPleshkova commented:
1. when you try to change smth here, the background and the pop up move a little bit (on one of my computers. on another-everything is ok)
![image](https://user-images.githubusercontent.com/8841018/63782295-da80e800-c8f3-11e9-88df-944106e14370.png)
2. Need to change up/down arrows there. Make these rectangle areas around them invisible and make the color of the arrows themselves #8997A5 

3. Start Abandoned from the capital letter
![image](https://user-images.githubusercontent.com/8841018/63804618-40836480-c920-11e9-944c-d292e4fd3362.png)
Btw, success modals look cool!

I guess these are the last tweaks from my side.

fernandoabolafio commented:
@MariaPleshkova thanks for your comments. Answering to each one specifically:

1. I wasn't able to reproduce it both on chrome or Firefox on my computer. Can you write an issue providing more details about how this happened? Like which browser you're using, which machine, etc. Some gifs/video of the issue would be nice too.

2. The number input component is currently being improved on pi-ui and should be updated in this repo soon enough: https://github.com/tiagoalvesdulce/pi-ui/pull/130.

3. Fixed ✅

MariaPleshkova commented:
@fernandoabolafio @tiagoalvesdulce 
it's chrome on windows, 1920x1080
![555](https://user-images.githubusercontent.com/8841018/64009271-3760e600-cb20-11e9-88ff-df9bf499a804.gif)
